huaweicloud/endpoints.go

```diff
@@ -33,22 +33,112 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Scope:            "global",
 		WithOutProjectID: true,
 	},
-
+	// ******* client for Compute start *******
 	"ecs": ServiceCatalog{
 		Name:    "ecs",
 		Version: "v1",
 	},
-	"compute": ServiceCatalog{
+	"computeV11": ServiceCatalog{
```

Comment:
suggest to add the catalog with lower-case

Reply:
okay, will do this in next patch.